tools/check-typo

```diff
@@ -159,6 +178,7 @@ EXIT_CODE=0
 ) | (
   while read f; do
     if test -n "$(check_prune "$f")"; then continue; fi
+    if $(git check-ignore -q "$f"); then continue; fi
```

Comment:
This change is logical in its own right - it's necessary for this GPR as otherwise executable checks are performed on built files (like `runtime/ocamlrun`).